### ISSUE TITLE
Blender BIM - Linework not printing m1 Mac

### ISSUE THREAD

**Issue by ChubbyQuark** [IfcOpenShell/IfcOpenShell#3325]
Lines don;t appear to be printing when creating a drawing, only fills. No errors when activating drawing. Same .ifc file prints ok from Windows. Currently using BB 230623: -

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/26485679/a998a380-9e28-4a68-ae3f-39ef96d61506)

Terminal output:

Initialize drawing generation process 0.00346491695381701
Generate underlay 1.745804911479354e-05
sync 0.1732732499949634
Processing body context 0.013250250020064414
Processing body context 3.3300602808594704e-07
Processing body context 2.08965502679348e-07
Processing body context 0.05274620896670967
Processing annotation context 0.012331040983553976
Processing annotation context 2.919696271419525e-07
Processing annotation context 1.6600824892520905e-07
Processing annotation context 0.011553958000149578
Finalizing 0.8576309999916703
Generate linework 1.3029524169978686
Generate annotation 0.0744421660201624
Combine SVG layers 0.016832416993565857
Drawing generation process 1.3978115000063553

**Comment by ChubbyQuark** [comment]
New cameras added to .ifc file work as intended, existing cameras/views aren't cutting linework or showing geometry

**Comment by Andrej730** [comment]
Same problem as #3055 and it was fixed in cb22b58 but still can be present in old .ifc models originated from windows.

The way to fix it is to either manually change paths in `EPset_Drawing` from backslashes to forward slashes or run script or to run script [replace_drawing_path.py](https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/blenderbim/scripts/replace_drawing_path.py) which will make those changes to all drawings in current .ifc model.

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/9417531/28d94b48-5cc1-420c-9f13-efd68f9f4dbd)


Also added some console warning in case BBIM couldn't find drawing's assets which can help with similar problems.
```
WARNING. Couldn't find Stylesheet for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\default.css. Drawing will be generated using default BBIM resource from /Users/m1/Library/Application Support/Blender/3.6/scripts/addons/blenderbim/bim/data/assets/drawings\assets\default.css
WARNING. Couldn't find Markers for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\markers.svg. Drawing will be generated using default BBIM resource from /Users/m1/Library/Application Support/Blender/3.6/scripts/addons/blenderbim/bim/data/assets/drawings\assets\markers.svg
WARNING. Couldn't find Symbols for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\symbols.svg. Drawing will be generated using default BBIM resource from /Users/m1/Library/Application Support/Blender/3.6/scripts/addons/blenderbim/bim/data/assets/drawings\assets\symbols.svg
WARNING. Couldn't find Patterns for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\patterns.svg. Drawing will be generated using default BBIM resource from /Users/m1/Library/Application Support/Blender/3.6/scripts/addons/blenderbim/bim/data/assets/drawings\assets\patterns.svg


WARNING. Couldn't find stylesheet for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\default.css
WARNING. Couldn't find markers for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\markers.svg
WARNING. Couldn't find symbols for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\symbols.svg
WARNING. Couldn't find patterns for the drawing by the path: /Users/m1/Desktop/drawing_bug/drawings\assets\patterns.svg
```
